main.go

```diff
@@ -71,33 +72,33 @@ func main() {
 	r.Handler("GET", *metricsPath, prometheus.Handler())
 
 	// Handlers for pushing and deleting metrics.
-	r.PUT("/metrics/job/:job/*labels", handler.Push(ms, true))
-	r.POST("/metrics/job/:job/*labels", handler.Push(ms, false))
-	r.DELETE("/metrics/job/:job/*labels", handler.Delete(ms))
-	r.PUT("/metrics/job/:job", handler.Push(ms, true))
-	r.POST("/metrics/job/:job", handler.Push(ms, false))
-	r.DELETE("/metrics/job/:job", handler.Delete(ms))
+	r.PUT(*prefixPath+"/metrics/job/:job/*labels", handler.Push(ms, true))
```

Comment:
We have a special wrapped router to do this in the Prometheus server, see https://github.com/prometheus/prometheus/blob/master/web/web.go#L167 . I'd suggest to do it in the same way here to avoid the many `*prefixPath` replications.